learn/why-ballerina/reliable-maintainable.md

```diff
@@ -93,26 +95,30 @@ This `is` operator-based error checking pattern is very common and you would end
 Ballerina provides a much more lightweight, shorthand for this pattern. The behavior of the function below is the same as the previous version but it is much more elegant. The `check expr` check expression performs an explicit error check, and the control flow also remains explicit.
 
 ```ballerina
+import ballerina/tcp;
+
 function whois(string domain, tcp:Client whoisClient) returns string|error {
-   // If `writeBytes` failed with an error, then `check` makes
-   //  the function return that error immediately
-   check whoisClient->writeBytes(domain.toBytes());
-   byte[] bytes = check whoisClient->readBytes();
-   return string:fromBytes(bytes);
+    // If `writeBytes` failed with an error, then `check` makes
+    //  the function to return that error immediately.
```

Comment:
Why do we need `to` here?

It's like

"make someone do something" vs "make someone to do something". @praneesha

Reply:
Sorry, my bad. +1 to remove `to`. @gimantha - Can you send the fix for this?